packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2306,24 +2306,39 @@ export const handlers = [
   }),
   http.get('*/monitor/services', () => {
     const response = {
-      data: [{ service_type: 'linode' }],
+      data: [
+        {
+          label: 'Linode',
+          service_type: 'linode',
+        },
+        {
+          label: 'Databases',
+          service_type: 'dbaas',
+        },
+      ],
     };
 
     return HttpResponse.json(response);
   }),
-  http.get('*/monitor/services/:serviceType/dashboards', () => {
+  http.get('*/monitor/services/:serviceType/dashboards', ({ params }) => {
     const response = {
-      data: [
+      data: [] as any[],
+    };
+    if (params.serviceType == 'linode') {
```

Comment:
```suggestion
    if (params.serviceType === 'linode') {
```

always use strict equality (===)

Reply:
Updated